identity/src/commonMain/kotlin/com/android/identity/credential/SecureAreaBoundCredential.kt

```diff
@@ -38,38 +38,46 @@ open class SecureAreaBoundCredential : Credential {
     /**
      * Constructs a new [SecureAreaBoundCredential].
      *
+     * [generateKey] providing [CreateKeySettings] must be called before using this object.
```

Comment:
Same as with MdocCredential, make the constructor private and have a static suspend create() that plays the role of a suspend constructor.